### DISCUSSION TITLE
Curso  Fullstack .NET - Dúvida sobre Minimal API

### DISCUSSION THREAD

**Discussion by CleidsonEstevam** [balta-io/forum#649] [Tira Dúvidas] [answered]
Opa, beleza? Vi que no curso Fullstack .NET é passado o conceito de Minimal API, mas as requisições são separadas em Handlers, nesse caso eu bato no método de EndPoint, e a partir dele bato no Handler onde esta a lógica para processar a requisição e devolver a resposta.

Muito se fala em quão simples tem que ser uma MinimalAPI por isso, minha dúvida é: Se usado assim, a MinimalAPI não se torna muito complexa (Deixando de ser "Minimal") ? 
Isto é, em um projeto de API padrão eu já não bateria bateria no EndPoint processaria e devolveria a informação para o cliente, com menos código e menos complexidade?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @CleidsonEstevam, como vai?

É complicado discutir o que é simples pois depende da sua ótica das coisas. Ao meu ver ficou simples, mas temos cursos que abordam só CQRS e outros que abordam só o MVC (Controllers) (Dica: Faça todos da carreira -> https://balta.io/carreiras)

Então em resumo, você tem que se sentir confortável com isto... se ainda não está, volta nos cursos anteriores, ou avança na carreira pra ter outras visões como DDD, e depois revisita o material.
